docs/_docs/02_features/05-catalog.md

```diff
@@ -23,13 +23,30 @@ terragrunt catalog <repo-url> [--no-include-root] [--root-file-name] [--output-f
 
 If `<repo-url>` is provided, the repository will be cloned into a temporary directory, otherwise:
 
-1. The repository list are searched in the config file `terragrunt.hcl`. if `terragrunt.hcl` does not exist in the current directory, the config are searched in the parent directories.
+1. The repository list are searched in the [config file](#configuration) `terragrunt.hcl`. if `terragrunt.hcl` does not exist in the current directory, the config are searched in the parent directories.
```

Comment:
```suggestion
1. The `urls` listed in the `catalog` configuration of your [parent configuration file](#configuration) file are used. If the root configuration file does not exist in the current directory, parent directories are recursively searched.
```

I know you didn't write most of this, but we're editing it here, so let's fix it.

Reply:
updated